### PR TITLE
google login added, design changed a bit, some fields + process  changed

### PR DESCRIPTION
**Pull request by ItsBhatt**

### CODE REVIEW

File: connectsync/src/components/Popup/PopupStyles.css [hunk 1]
```diff
@@ -11,10 +20,10 @@
   opacity: 0.5;
 }
 
-.modal-content {
+/* .modal-content {
```
Comment (jawadsab) [inline connectsync/src/components/Popup/PopupStyles.css:23]: the popup is going beyond the screen. So in the modal content overflow-y:auto; max-height: 95vh would be better in my opinion.

File: connectsync/src/components/Popup/Popup.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const Popup = forwardRef((props, ref) => {
         <div className="modal-backdrop"></div>
         <div className="modal-dialog w-75">
           <div className="modal-content p-5">
-            <div className="border-0 p-0">
+            <div className="border-0 p-0 position-absolute close-btn">
```
Comment (yewyewXD) [inline connectsync/src/components/Popup/Popup.js:30]: isn't the close button already there?

Comment (ItsBhatt) [reply connectsync/src/components/Popup/Popup.js:30]: it was already there, just added class for styling

File: connectsync/src/components/Popup/PopupStyles.css (hunk 1 above)
Comment (ItsBhatt) [reply connectsync/src/components/Popup/PopupStyles.css:23]: popup shouldn't be scrollable inside

**Pull request merged**